django/utils/autoreload.py

```diff
@@ -431,8 +442,15 @@ def _get_clock(self, root):
 
     def _subscribe(self, directory, name, expression):
         root, rel_path = self._watch_root(directory)
+        # Only receive notifications of files changing, filtering out other types
+        # like special files: https://facebook.github.io/watchman/docs/type
+        only_files_expression = [
+            "allof",
+            ["anyof", ["type", "f"], ["type", "l"]],
+            expression
+        ]
         query = {
-            'expression': expression,
+            'expression': only_files_expression,
```

Comment:
Is this a separate change? (I've left it for now but... 🤔)

Reply:
I forget the context, but I remember it being required for it to work locally. 🤔

Reply:
So definitely part of this change. No problem. (Didn't know if it should be there anyway...) (Not a biggie)